src/components/ParticipantList/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Icon } from "antd";
+import "./style.css";
+
+const Heading = ({ children }) => {
+  return <h2 className="heading">{children}</h2>;
+};
+const ParticipantListItem = ({ participant, onAddParticipant }) => {
+  return (
+    <tr key={participant.id}>
+      <td className="dark">{participant.name}</td>
+      <td className="dark">{participant.surname}</td>
+      <td className="dark right">{participant.examPoints}</td>
+      <td className="centered">
+        <Icon
+          onClick={() => onAddParticipant(participant.id)}
+          type="plus"
+          className="action"
+        />
+      </td>
+    </tr>
+  );
+};
+
+export class ParticipantList extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.containerRef = React.createRef();
+    this.state = {
+      offsetTop: 0
+    };
+  }
+
+  componentDidMount() {
+    const container = this.containerRef && this.containerRef.current;
+    const offsetTop = container.offsetTop;
+    this.setState({
+      offsetTop
+    });
+  }
+
+  render() {
+    const { city, participants, onAddParticipant } = this.props;
+    return (
+      <div
+        className="container"
+        ref={this.containerRef}
+        style={{ minHeight: `calc(100vh - ${this.state.offsetTop}px)` }}
+      >
+        <div>
+          <Heading>{`All Students (${city})`}</Heading>
+        </div>
+        {participants.length ? (
+          <table className="list">
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Surname</th>
+                <th className="right">Test result</th>
+                <th width="150" className="centered">
+                  Add to my group
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              {participants.map(participant => (
+                <ParticipantListItem
+                  key={participant.id}
+                  participant={participant}
+                  onAddParticipant={onAddParticipant}
+                />
+              ))}
+            </tbody>
+          </table>
```

Comment:
I wonder about using table here. On one hand, it seems ok in the sense that, well, you're presenting tabular data. On the other... tables in HTML have... problems. I'm not saying that it's necessarily a mistake, but sticking to divs would probably make things easier to style and adjust in the long run.
Then again, we're clearly dealing with tabular data here so I'm not saying you made a mistake in your choice of HTML tags here.